vault/logical_passthrough.go

```diff
@@ -126,14 +126,13 @@ func (b *PassthroughBackend) handleRead(
 	}
 
 	// Check if there is a ttl key
-	var ttl string
-	ttl, _ = rawData["ttl"].(string)
-	if len(ttl) == 0 {
-		ttl, _ = rawData["lease"].(string)
-	}
 	ttlDuration := b.System().DefaultLeaseTTL()
-	if len(ttl) != 0 {
-		dur, err := parseutil.ParseDurationSecond(ttl)
+	ttlInt, ok := rawData["ttl"]
```

Comment:
Small nit, but I interpreted this variable as "TTL integer" when i think it's supposed to be "TTL interface", could you rename it to `ttlRaw` or something similar?